src/main/scala/MondayScala.scala

```diff
@@ -1,3 +1,36 @@
 object MondayScala extends App {
+  //Hello World!
+  println("Hello World")
+  //Assignment
+  val helloW = println("Hello World!")
+  helloW
+  //Parameters
+  def stringMethod(a: String)= {
+    println(a)
+  }
+  stringMethod("This is a test")
+  //Return Types
+  def add( num1:Int, num2:Int ) : Int = {
+    var sum:Int = 0
+    sum = num1 + num2
+    println(sum)
+    sum
+  }
+  add(12, 12)
+  //Type Inference
+  def parameterInference(c: Any) = c match {
```

Comment:
you don't have a vildcard case, anything outside your 4 cases comes in and it breaks